src/main/webapp/data/userMapData.json

```diff
@@ -203,6 +205,7 @@
       "Simon Fraser University",
       "Southern Alberta Institute of Technology",
       "St. Andrew's College",
+      "St. Barbara Catholic Elementary School",
```

Comment:
Drop the `School` suffix for consistency with `St. David Catholic Elementary` below?